spec/appendix.md

```diff
@@ -1,12 +1,58 @@
 ## Appendix
 
+### Trust Over IP Glossary - Controlled Terms 
+
+NOTE: 
+* The following set of identifiers are from the current [ToIP Glossary](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit?usp=sharing) (Google Doc). These definitions need to be moved over to the (under development) ToIP Terms tool (aka TEv2).
```

Comment:
It’s not clear what definitions “need to be moved” — the ones in this spec or the ones in the ToIP Glossary.  I think you mean that the “ToIP Glossary definitions are being moved to the (under development) ToIP Terms tool (aka TEv2). 

Suggest you clarify.

Reply:
That's going to be an ongoing effort as we edit this (and other specs).

Reply:
relates to this Issue: https://github.com/trustoverip/tswg-did-method-webs-specification/issues/116

Reply:
@darrellodonnell I believe this should be moved out into a separate issue and PR (its basically a TODO)